docs/_edot-sdks/features.yml

```diff
@@ -10,7 +10,7 @@ features:
       min_version: 1.0
     Node.js:
       status: ga
-      min_version: 1.0
+      min_version: "0.2.0"
```

Comment:
review note: I'm fine moving this back to 1.0 if the aesthetics are better in the table. No one is ever going to use old version 0.2.0.

Reply:
yes, let's move it back to 1.0 (I think we can ignore the pre-GA versions in this table)